
C:\Users\Hello World!\Desktop\COMPLETE\RPA>call python rpa.py 
Error checking for updates: GitLab repository not reachable.
Starting application normally...
 * Serving Flask app 'rpa'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5213
 * Running on http://192.168.1.12:5213
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 122-264-443
 * Detected change in 'C:\\Users\\Hello World!\\Desktop\\COMPLETE\\RPA\\rpa.py', reloading
Error checking for updates: GitLab repository not reachable.
Starting application normally...
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 122-264-443
 * Detected change in 'C:\\Users\\Hello World!\\Desktop\\COMPLETE\\RPA\\rpa.py', reloading
Error checking for updates: GitLab repository not reachable.
Starting application normally...
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 122-264-443
Error checking for updates: GitLab repository not reachable.
Starting application normally...
Terminate batch job (Y/N)? 
C:\Users\Hello World!\Desktop\COMPLETE\RPA>call python rpa.py 
Error checking for updates: GitLab repository not reachable.
Starting application normally...
 * Serving Flask app 'rpa'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5213
 * Running on http://192.168.1.12:5213
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 122-264-443
Error checking for updates: GitLab repository not reachable.
Starting application normally...
Terminate batch job (Y/N)? 